openapi.yaml

```diff
@@ -16,176 +16,405 @@ host: api.oregonstate.edu
+      operationId: patchGameByGameId
+      consumes:
+        - application/json
+      parameters: 
+        - $ref: '#/parameters/gameId'
+        - name: updateBody
+          in: body
+          description: Object that will overwrite the game specified by its id
+          required: true
+          schema:
+            $ref: '#/definitions/gamePatchBody'
+      responses:
+        '200':
+          description: Successfully update a game
+          schema:
+            $ref: '#/definitions/gameResult'
         '400':
-          description: Bad request
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/errors'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+    delete:
+      tags:
+      - games
+      summary: "Remove a game specified by its id"
+      operationId: deleteGameByGameId
+      parameters:
+        - $ref: '#/parameters/gameId'
+      responses:
+        '204':
+          description: game successfully removed
+        '404':
+          description: No game with this id
           schema:
-            $ref: '#/definitions/Errors'
+            $ref: "#/definitions/error"
         '500':
           description: Internal server error
           schema:
-            $ref: '#/definitions/Errors'
-  /pets/{id}:
+            $ref: "#/definitions/error"
+  /games/{gameId}/players:
     get:
-      summary: Info for a specific pet
+      summary: A list of player in the game with their specific in-game information
       tags:
-        - pets
-      description: Get pet by unique ID
-      operationId: getPetById
+        - players
+      operationId: getPlayersByGameId
       parameters:
         - $ref: '#/parameters/authorization'
-        - name: id
-          in: path
-          description: Unique Pet ID
+        - $ref: '#/parameters/gameId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/playerInGameResults'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/errors'
+  /games/{gameId}/players/{playerId}:
+    get:
+      summary: Info of the player in terms of the specific game
+      tags:
+        - players
+      operationId: getPlayerByGameIdAndPlayerId
+      parameters: 
+        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/playerId'
+        - $ref: '#/parameters/gameId'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/playerInGameResult'
+        '404':
+          description: Player not found
```

Comment:
Since a 404 could also mean that the game was not found, you should specify this in the description